### PR TITLE
docs: add ADR referencing OEP-58

### PR DESCRIPTION
**Pull request by Carlos-Muniz**
This PR adds the first ADR. It references OEP-58 as the reason for establishing this repo.

### CODE REVIEW

File: decisions/0001-translations-management.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#####################################################
+Establishing a Repository for Translations Management
+#####################################################
+
+***********
```
Comment (feanil) [inline decisions/0001-translations-management.rst:5]:
```suggestion
```

File: decisions/0001-translations-management.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#####################################################
```
Comment (feanil) [inline decisions/0001-translations-management.rst:1]:
```suggestion
```
You don't need the Overlines.

**Review by feanil — APPROVED** [review]
This file should go in a `docs/decisions` folder. Do that, drop the overlines and then I think this is good to land.

**Review by sarina — APPROVED** [review]

File: docs/decisions/0001-translations-management.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Establishing a Repository for Translations Management
+#####################################################
+
+Description
+***********
+The openedx-translations repository has been established as part of `OEP-58`_. This
+repository will contain the Translations files for all Open edX repositories. For more
+information, please read `OEP-58`_.
+
+.. _OEP-58: https://github.com/openedx/open-edx-proposals/pull/367
```
Comment (sarina) [inline docs/decisions/0001-translations-management.rst:10]: Make a note to come back and fix this link to point to the RTD page.

**Pull request merged**